docs/reference/sso.md

```diff
@@ -0,0 +1,522 @@
+
+Steps:
+1. In the Azure portal, in App registrations, select your application.
+2. Select Certificates & secrets > Client secrets > New client secret.
+3. Add a description for your client secret.
+4. Select an expiration for the secret or specify a custom lifetime.
+    1. Client secret lifetime is limited to two years (24 months) or less. You can't specify a custom lifetime longer than 24 months.
+    2. Microsoft recommends that you set an expiration value of less than 12 months.
+5. Select Add.
+6. Record the secret's value for use in your client application code. This secret value is never displayed again after you leave this page.
+
+#### Add a redirect URI
+A redirect URI is the location where the Microsoft identity platform redirects a user's client and sends security tokens after authentication.
+
+You add and modify redirect URIs for your registered applications by configuring their platform settings.
+
+Enter https://lakefs-cloud.us.auth0.com/login/callback or https://lakefs-cloud.eu.auth0.com/login/callback (depends on your organization data location) as your redirect URI.
+
+Settings for each application type, including redirect URIs, are configured in Platform configurations in the Azure portal. Some platforms, like Web and Single-page applications, require you to manually specify a redirect URI. For other platforms, like mobile and desktop, you can select from redirect URIs generated for you when you configure their other settings.
+
+Steps:
+1. In the Azure portal, in App registrations, select your application.
+2. Under Manage, select Authentication.
+3. Under Platform configurations, select Add a platform.
+4. Under Configure platforms, select the web option.
+5. Select Configure to complete the platform configuration.
+
+Once you finish registering lakeFS Cloud with Azure AD, save the **Application (Client) ID**, **Application Secret Value** and send this to Treeverse's team to finish the integration.
+
+## Enterprise
+
+lakeFS Enterprise provides a secondary service, running side-by-side with lakeFS which handles the authentication, this service is called Fluffy.
+Once you've onboarded to [lakeFS Enterprise](../enterprise.md) and have access to the authentication service (Fluffy), you can configure one of the supported authentication methods below.
+lakeFS Enterprise leverage Helm's capabilities to run the sidecar with it's configuration, on top of the specific-authentication method, please review the [Helm configuration](#helm).
+
+### AD FS (using SAML)
+{: .d-inline-block }
+Enterprise
+{: .label .label-purple }
+
+> **Note**: AD FS integration is using certificates to sign & encrypt requests going out from Fluffy and decrypt incoming requests from AD FS server.
+
+In order for fluffy to work, some values must be configured, update the relevant values section in this file and comment it out.
```

Comment:
I don't understand what `this file` refers to, and why would you update them and then comment them out? 

```suggestion
In order for fluffy to work, the following values must be configured. Update the relevant values section in this file and comment it out.
```

Reply:
fixed, thanks